### DISCUSSION TITLE
The mod should be able to copy curse scroll as well

### DISCUSSION THREAD

**Discussion by bakaemon** [HunterGhost27/Scribe-Scrolls-and-Skillbooks#5] [Ideas]
As I tried to become non-mage spellcasters, it's saddened me as the fact that there is only scroll I can't scribe: Curse. Additionally, other non-elemental spell (there is a mod that add mass bless, mass purge, .etc) also can not be scribed. I suggest that non-elemental scroll should be also implemented without the use of essence (or just use shadow essence if you like).
I hope that the next update will be there soon, best of luck!

**Comment by Shresht7** [comment]
You **can** scribe Curse scrolls; you need ***Alien Tormented Souls***. `Curse Scroll + Alien Tormented Soul + Sheet-of-Paper + Inkpot-&-Quill`

The mod generates scribing recipes *dynamically* (i.e. on runtime) based on your modlist. If a mod allows you to craft a scroll, then you can probably scribe it. If the mods that add new scrolls (along with their recipes) follow Larian's conventions, then they should be compatible. However, I cannot guarantee that this will work for every mod out there.

The update is basically done. Just need to test it a little before deploying.
